### ISSUE TITLE
Reading event before latest one

### ISSUE THREAD

**Issue by salvq** [vincentcox/QNAP-pushover#2]
Seems that the script is reading and sending event before the latest. Every time I simulate event I got the previous one.

I have been trying the latest commit from [mtlott](https://github.com/mtlott/QNAP-pushover)

What parameter to change in python script to read one ahead event ?

Pushover event
![image](https://user-images.githubusercontent.com/43242348/97618814-cd8fe500-1a1f-11eb-83f7-33efae7411c4.png)

qnap event
![image](https://user-images.githubusercontent.com/43242348/97618756-b94be800-1a1f-11eb-897f-55dc67eb1597.png)

**Comment by salvq** [comment]
I have made further troubleshooting and find out the reading is always triggered by the new event but reading is always one before the actual / latest one

Event from 20:01:42 was sent (triggered by python) when new event from 20:02:34 was created...
![image](https://user-images.githubusercontent.com/43242348/97620568-0cbf3580-1a22-11eb-9f9a-1efa614abaca.png)

**Comment by vincentcox** [comment]
hi @salvq,
I suspect that this can be fixed by modifying this [line](https://github.com/vincentcox/QNAP-pushover/blob/master/main.py#L52).
Try to edit it to:
```
"SELECT * FROM " + DB_NAME + " where event_id="+str(LATEST_EVENT_ID-i)+1";")
```

**Comment by salvq** [comment]
I am getting an error while executing the change...

```
/ # python main.py
  File "main.py", line 52
    "SELECT * FROM " + DB_NAME + " where event_id="+str(LATEST_EVENT_ID-i)+1";")
                                                                              ^
SyntaxError: invalid syntax
/ #

```

**Comment by TheCatLady** [comment]
@vincentcox I've been working on my own fork of this project, so I hope you don't mind me chiming in.

@salvq Not sure if you ever figured it out, but if not you can try
```
            cursor.execute(
                "SELECT * FROM " + DB_NAME + " WHERE event_id="+str(LATEST_EVENT_ID-i+1)+";")
```

That said, I've added several improvements (HTML formatting, notification priority & timestamps, persisting last-processed event, filtering, etc.) if you want to check out my repo: https://github.com/TheCatLady/docker-qnap-pushover

Would love to get feedback from the both of you! 😄

**Comment by vincentcox** [comment]
@TheCatLady: I just checked it out, **it works like a charm**.

I tested it with locking my phone out, by entering multiple times the incorrect password via ssh:

![image](https://user-images.githubusercontent.com/9286611/104289938-00436880-54ba-11eb-8e3b-1fea6c51ce13.png)

I'll archive this repo later this day and edit the `readme` file to point to your repo. I'm happy that someone resurrected the original idea and took it to the next level. (You did a great job with the documentation!)

@salvq: I really recommend checking her repo out.

Have an awesome day all!
